.gitignore

```diff
@@ -1 +1,2 @@
-.azure
\ No newline at end of file
+.azure
+.env
```

Comment:
Most of these files are missing newlines at end of files, its nice to add those. I use pre-commit to enforce that in my repos.

Reply:
Updated. Thanks for the recommendation.